CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Puter
+
+Welcome to Puter, the open-source distributed internet operating system. We're excited to have you contribute to our project, whether you're reporting bugs, suggesting new features, or contributing code. This guide will help you get started with contributing to Puter in different ways.
+
+<br>
+
+# Report bugs
+
+Before reporting a bug, please check our [the issues on our GitHub repository](https://github.com/HeyPuter/puter/issues) to see if the bug has already been reported. If it has, you can add a comment to the existing issue with any additional information you have.
+
+If you find a new bug in Puter, please [open an issue on our GitHub repository](https://github.com/HeyPuter/puter/issues). We'll do our best to address the issue as soon as possible. When reporting a bug, please include as much information as possible, including:
```

Comment:
I suggest to either guide the contributor to apply the bug label here, or communicate to contributors, that maintainers will do that classification for them to disambiguate that flow.

Also if there will be a bug/issue template provided(as @AtkinsSJ) suggested, it should link to that as well. A good example would be [osu's bug issue template](https://www.github.com/ppy/osu/blob/master/.github/ISSUE_TEMPLATE/bug-issue.yml) (MIT licensed if that's of concern).

Reply:
Great idea. We're going to label it for them for now so that we get the hang of the appropriate labels. I'm going to clarify this in the guide.

> Also if there will be a bug/issue template provided(as @AtkinsSJ) 

Thanks for the suggestions. Looking into it.

Reply:
@AtkinsSJ @lsileoni Decided to hold off on enabling issue templates for the next one month. I want us to get a feel of what our template should look like based on the first 50-100 issues filed. I put a reminder in my calendar to get back to this in a month.